packages/runtime/container-runtime/src/test/summarizerClientElection.spec.ts

```diff
@@ -174,12 +174,15 @@ describe("Summarizer Client Election", () => {
 
     function assertState(
         expectedId: string | undefined,
+        expectedParentId: string | undefined,
```

Comment:
Not sure if there was a reason for Paul not to add the ParentId now that we serialize it, so I went ahead and added it to the AssertState.